### ISSUE TITLE
Fabric version needed

### ISSUE THREAD

**Issue by zhenlige** [Cadiboo/NoCubes#91]
Since Fabric is a more modern lightweight modloader and is prefered by most new developers, it is a good choice to transplant the mod to Fabric.

**Comment by Cadiboo** [comment]
Yep, I’m moving towards fabric compatibility (note the migration from JS ASM to Mixin). Progress is slow and I’m unfamiliar with Fabric - any contributions would be appreciated.
